apps/metrics/views.py

```diff
@@ -1,10 +1,41 @@
+from django.db.models import Count
+from django.contrib.auth.models import User
+from oauth2_provider.models import AccessToken
+from rest_framework.serializers import BooleanField, CharField, IntegerField, ModelSerializer
+from rest_framework.generics import ListAPIView
+from rest_framework.pagination import PageNumberPagination
 from rest_framework.permissions import IsAuthenticated, IsAdminUser
-from rest_framework.renderers import JSONRenderer
+from rest_framework.renderers import JSONRenderer, BrowsableAPIRenderer
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from ..accounts.models import UserProfile
 from ..dot_ext.models import Application
-from oauth2_provider.models import AccessToken
+
+
+class UserSerializer(ModelSerializer):
+
+    class Meta:
+        model = User
+        fields = ('id', 'username', 'email')
+
+
+class AppMetricsSerializer(ModelSerializer):
+
+    app_id = IntegerField(source='id', read_only=True)
+    app_name = CharField(source='name', read_only=True)
+    app_active = BooleanField(source='active', read_only=True)
+    dev_user = UserSerializer(source='user', read_only=True)
```

Comment:
these should be under their normal keys `id, name, active, user`. (might not need `active` atm?)